src/test/java/org/jvnet/hudson/plugins/shelveproject/ShelveProjectActionTest.java

```diff
@@ -21,37 +24,39 @@
 public class ShelveProjectActionTest {
     @Rule
     public JenkinsRule jenkinsRule = new JenkinsRule();
+    private FreeStyleProject project = null;
```

Comment:
nit: we don't care about it being a FreeStyleProject:

```suggestion
    private Item project;
```

(you'll also have to modify the imports)